### ISSUE TITLE
Formatter configuration

### ISSUE THREAD

**Issue by gonnaduobitofhelping** [duolicious/duolicious-frontend#296]
Currently there isn't a formatter installed and configured in the project. I assume the code is currently formatted by a formatter configured locally in the IDE of @duogenesis. The IDE-s of other contributors can however be configured differently, overriding the existing formatting. I suggest installing Prettier (or another formatter of preference) in the project and configuring it with conventions that are currently implicitly used as this would improve the DX of other contributors.

**Comment by duogenesis** [comment]
This is a good idea. A similar thing I wanted to add was type checking in CI (https://github.com/duolicious/duolicious-frontend/issues/297).

**Comment by DeepeshKalura** [comment]
@duogenesis  Ok I understand the issue I will looks some modern linter for the web and put the analysis of each of them here and then we can choose one linter

**Comment by DeepeshKalura** [comment]
@duogenesis  Yeah after extensive lazy research, I come to the conclusion we can make a configuration for the  EsLinter 

https://medium.com/@deepeshkalura/modern-web-linter-and-why-we-choose-eslinter-a5d3c2354a2a

advantages explain In this medium post 

so,  this is the https://kadikraman.github.io/react-native-v2/code-style

this is the docs https://eslint.org/docs/latest/use/getting-started

sounds good to you ??  and more more thing What do you mean by  "I wanted to add was type checking in CI #297 "  Does this mean you have Continues Integration and you want a linter check like people put types o not

**Comment by duogenesis** [comment]
@DeepeshKalura

>sounds good to you ??

Sounds great

>Does this mean you have Continues Integration and you want a linter check like people put types o not

Pretty much. Actually missing types are fine for now. I just want to catch type errors

**Comment by DeepeshKalura** [comment]
@duogenesis  Assign me this task to me I will complete it by today.

I will add  EsLinter to the React native. In this pr not going to touch ci type checking I will next pr of this.  

sound good ??

**Comment by duogenesis** [comment]
@DeepeshKalura Sounds good

**Comment by duogenesis** [comment]
@DeepeshKalura haha I've formally assigned you now. Although you can basically pick up any ticket and nobody's going to steal it from you. There's not many contributors right now.

**Comment by DeepeshKalura** [comment]
> @DeepeshKalura Haha I've formally assigned you now. Although you can basically pick up any ticket and nobody's going to steal it from you. There are not many contributors right now.

Nah!  It is not going to steel feel but rather I want you to enjoy coding with me, man! In my mind  I was thinking it can be good assured for you at I am coding with someone.  

Anyway, I will make a website would you like me to give me a review which thing can make it good and also faster build time?  I am need to make a website which showcases book of my sister. Additionally I want him to write the book in markdown so we can better format next time ( word give me heatattack to format ).  So, I was thinking to suggest obsidian for her where he can write some book when see finish arc or chapter then automatically she can read and mail get publish to the one I was thinking to create this sort of file. 

If have suggestion let me know 

After that I will do your project.

**Comment by DeepeshKalura** [comment]
@duogenesis  Damm 5000 error feel like dead man to me

![image](https://github.com/user-attachments/assets/405f3a29-08b7-46e6-99f3-3a62bf0861b6)

**Comment by DeepeshKalura** [comment]
> @duogenesis Damm 5000 error feel like dead man to me
> 
> ![image](https://private-user-images.githubusercontent.com/99230888/402100269-405f3a29-08b7-46e6-99f3-3a62bf0861b6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY2MTM1NzEsIm5iZiI6MTczNjYxMzI3MSwicGF0aCI6Ii85OTIzMDg4OC80MDIxMDAyNjktNDA1ZjNhMjktMDhiNy00NmU2LTk5ZjMtM2E2MmJmMDg2MWI2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTExVDE2MzQzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVhMGUyNTkxNzc0OTEyMDgxYjY3ZmNhZWZjYzU5YzQ3MDE1MDA5OWM1N2M3YTBlOTk2MTM3MmUxYTY2YTExOWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0._A4R7CS6dSKsbdu-yFcyvCBNlGj-cmovlk8rz2Eta9s)

@duogenesis  did you miss this
